content/2021/2021-01-22_moderators/index.md

```diff
@@ -0,0 +1,18 @@
++++
+title = "Codewars moderation team"
+description = "Codewars staff reinforced by a team of community members!"
+
+[taxonomies]
+tags = ["announcement", "moderation", "community"]
++++
+
+
+Until recently, Codewars maintainers and admins were overwhelmed by amount of tasks related not only to the maintenance of the platform, fixing bugs, and adding new features, but also by issues related to Codewars content and users (poor **@Kazk** \*hug\*). Sometimes it was just too much for them.
+
+To take some of the burden off of their backs, a group of trusted community members has been selected and granted a title of **moderator**, forming the community support team: a team which would help admins handling some tasks related to users and content. Thanks to recent refactorings it was finally possible to introduce additional tools and features which can be used by priviledged members to perform additional tasks.
+
+Moderators are meant to be the first line of support for Codewars users and intermediaries between community, and administrators. They have access to tools which help them with handling bad quality content, problematic and dishonest users, they can also manage Codewars repositories used for tracking [documentation](https://github.com/codewars/docs) or [content issues](https://github.com/codewars/content-issues/issues). Whenever a new moderation feature will be added which would be in some way helpful for regular users, we will announce it with a dedicated post.
```

Comment:
```suggestion
Moderators are meant to be the first line of support for Codewars users and intermediaries between the community and administrators. They have access to tools that help them with handling bad-quality content and problematic and dishonest users, they can also manage Codewars repositories used for tracking [documentation](https://github.com/codewars/docs) or [content issues](https://github.com/codewars/content-issues/issues). Whenever a new moderation feature will be added which would be in some way helpful for regular users, we will announce it with a dedicated post.
```

What about mentioning about for those who joined the community-support team, they can also manage issues on the codewars.com repo?

Reply:
I am not sure if its worth mentioning specifically here in the announcement, but i will add a note to "moderator features" reference page.